src/urls.py

```diff
@@ -100,28 +105,28 @@
             name="game_network_groups"
         ),
         path(
-            "frontend/training-games/<run>/<network>/",
-            GamesListView.as_view(),
+            "frontend/training-games/<run>/by-network/<network>/",
+            GamesListByNetworkView.as_view(),
             {"kind": "training"},
-            name="training_games_list"
+            name="training_games_list_by_network"
         ),
         path(
-            "frontend/training-games/<run>/<network>/<page>/",
-            GamesListView.as_view(),
+            "frontend/training-games/<run>/by-network/<network>/<page>/",
+            GamesListByNetworkView.as_view(),
             {"kind": "training"},
-            name="training_games_list"
+            name="training_games_list_by_network"
         ),
         path(
-            "frontend/rating-games/<run>/<network>/",
-            GamesListView.as_view(),
+            "frontend/rating-games/<run>/by-network/<network>/",
```

Comment:
Iwill say that the url that makes more sense is 

`frontend/network/<network>/rating-games/` (not sure if frontend is nice to have in the url, by the way.)

Reply:
Thanks, got rid of "frontend".
What would you propose for the replacement for "training-games/<run>/by-user/<user>/"  then? We would want to keep it consistent.

Reply:
Okay I set a new URL scheme that is extremely similar to what you proposed. Thanks!